### DISCUSSION TITLE
Guest system crash wipes HDD image from SD card

### DISCUSSION THREAD

**Discussion by ghost** [BlueSCSI/BlueSCSI-v2#183] [Help]
In HP-UX 10.20, while exploring and testing the system's limits, a system crash with coredump was triggered and the system was halted for a hard reboot. Unfortunately, the related HDD image together with a folder containing multiple ISO images was wiped from the SD card in the process. Also both the bluescsi.ini and the log.txt were deleted. This happened twice for this OS.

In NeXTSTEP 3.3 a similar crash occurred while copying the contents of a CD-ROM (=ISO image) to a secondary HDD (=HD image) was performed. The system halted and a dialog window offered rebooting. Here also, the systems HD image along with the bluescsi.ini and the log.txt were wiped from the SD card. This happened once, so far.

* Hardware this occurred on is HP 9000 715/64 (PA-RISC).
* Using a BlueSCSI v2 Centronics (revision 2023/10a) with firmware version v2024.05.21 
* Operating systems on HDD images: HP-UX 10.20 and NeXTSTEP 3.3, both imaged from physical disks.

Of course, without any log.txt there is hardly any suitable information for debugging available.  Unfortunately, i didn't make any photo of the display showing the OS error messages. 

Please take this report as a preliminary heads-up until the described crash situations can be reproduced in order to take photos for a followup.

Thanks

**Comment by erichelgeson** [comment]
Thanks for the report. I'd start with a full format with the SD Card Formatter Utility - it can resolve 99% of issues to do with SD cards. Also go through this guide to rule out easy things first https://github.com/BlueSCSI/BlueSCSI-v2/wiki/Troubleshooting#what-to-try-first.
